presto-hive-hadoop2/bin/common.sh

```diff
@@ -132,3 +132,12 @@ function start_hadoop_docker_containers() {
   # wait until hadoop processes is started
   retry check_hadoop
 }
+
+function get_hive_major_version() {
+    local version=$(exec_in_hadoop_master_container hive --version 2>/dev/null | sed 's/^Hive.*[ ]\([0-9]\)\..*/\1/;t;d')
```

Comment:
`local` must be on sep line, otherwise it masks exit codes

what does `t` and `d` mean?

btw my idiom for grep+sed is
```
sed -n 's/..../..../p'
```

Reply:
https://stackoverflow.com/a/1665662.

DIfferent method. Same result.